### ISSUE TITLE
Date formating for outings

### ISSUE THREAD

**Issue by asaunier** [c2corg/v6_ui#245]
I had some precision from the client for outings dates (can be managed in a separated PR if you prefer):
* [x] date to show in outings lists is ``date_end``
* [x] showing the date as subtitle of outings detail view page would be nice to make the date more obvious
* [x] if date_end != date_start, if possible it would look like:
```
12 - 16 mars 2016
26 mars - 3 avril 2016
27 décembre 2015 - 4 janvier 2016
```

The client asked if adding the weekday ("dimanche") would be (reasonably!) possible? Perhaps with momentjs?

**Comment by ginold** [comment]
it is possible but i don't see having such a long date. lundi 27 décembre 2015 - 4 vendredi janvier 2016, longer than the title :P

**Comment by tsauerwein** [comment]
Maybe show the short format and set the long format in the title attribute, so that if you hover the data you can see the weekdays.

**Comment by asaunier** [comment]
Fixed by #266
